src/clj/game/core/flags.clj

```diff
@@ -36,7 +36,8 @@
   "Returns true if the runner is tagged."
   [state]
   (or (pos? (get-in state [:runner :tag]))
-      (pos? (get-in state [:runner :tagged]))))
+      (pos? (get-in state [:runner :tagged]))
```

Comment:
Shouldn't `:tagged` be a boolean instead of a number?

Reply:
This is not my change, it's been this way ever since Paparazzi.

Reply:
I shall rectify this at some point